docs/usage/istio.md

```diff
@@ -6,6 +6,10 @@
 
 When enabled in gardenlet the `ManagedIstio` feature gate can be used to deploy a Gardener-tailored Istio installation in Seed clusters. It's main usage is to enable features such as [Shoot API server SNI](../proposals/08-shoot-apiserver-via-sni.md). This feature should not be enabled on a Seed cluster where Istio is already deployed.
 
+However, this feature gate is deprecated, turned off by default and will be removed in a future version of Gardener.
```

Comment:
```suggestion
However, this feature gate is deprecated, turned on by default and will be removed in a future version of Gardener.
```

Reply:
At least, it looks like it is enabled by default. The following sentence also suggests that it is not turned off.

Reply:
Nice catch :D